nibabel/tests/test_nifti1.py

```diff
@@ -13,17 +13,18 @@
 
 import numpy as np
 
-from ..externals.six import BytesIO
-from ..casting import type_info, have_binary128
-from ..tmpdirs import InTemporaryDirectory
-from ..spatialimages import HeaderDataError
-from ..eulerangles import euler2mat
-from ..affines import from_matvec
-from .. import nifti1 as nifti1
-from ..nifti1 import (load, Nifti1Header, Nifti1PairHeader, Nifti1Image,
-                      Nifti1Pair, Nifti1Extension, Nifti1Extensions,
-                      data_type_codes, extension_codes, slice_order_codes)
-
+from nibabel import nifti1 as nifti1
```

Comment:
I thought the standard is to use absolute imports in tests. At least, @effigies and I have been migrating to that standard as we touch test files.